--- old/CMakeLists.txt	2015-10-19 19:14:01.979902478 +0200
+++ new/CMakeLists.txt	2015-10-19 19:15:04.355911652 +0200
@@ -68,7 +68,7 @@
     set(_lua_module_dir "${_lua_lib_dir}/lua/5.1")
 endif()
 
-add_library(cjson MODULE lua_cjson.c strbuf.c ${FPCONV_SOURCES})
+add_library(cjson STATIC lua_cjson.c strbuf.c ${FPCONV_SOURCES})
 set_target_properties(cjson PROPERTIES PREFIX "")
 target_link_libraries(cjson ${_MODULE_LINK})
 install(TARGETS cjson DESTINATION "${_lua_module_dir}")
